code/index.html

```diff
@@ -3,28 +3,193 @@
+            <div></div>
+        </div>
+
+        <h1>Plant Film Festival</h1>
+
+    </header>
+    
+    <!-- Content-wrapper contains all content of the website, it has grid column layout -->
+    <div class="content-wrapper">
+
+      <!-- One of two content-boxes, FAQ section -->
+      <section id="faq-container" class="content-box">
+          <div class="headline"><h1>I want to watch plants grow...</h1></div>
+  
+          <div class="question" id="section1">How can I find the schedule of screenings?<span class="arrow">+</span></div>
+          <div class="answer">
+          <p>You can find our full live events schedule in our program as well as on our interactive event grid PDF.</p>
+          </div>
+  
+          <div class="question" id="section2">When can I start watching?<span class="arrow">+</span></div>
+          <div class="answer">
+          <p>Most films and programs will be available from 12:01am on April 8, 2022 until 11:59pm PT on April 18, 2022. Select events and films have set viewing times and dates noted within the digital program and on the website.</p>
+          </div>
+
+          <div class="question" id="section3">How long do I have to watch a film that is not a timed event?<span class="arrow">+</span></div>
+          <div class="answer">
+          <p> For films that are not timed events, passholders can watch the film until the set number of views are reached.</p>
+          </div>
+
+          <div class="question" id="section4">Can I watch the films from anywhere?<span class="arrow">+</span></div>
+          <div class="answer">
+          <p> Film viewing is limited to the EU. All items are geo-blocked so you won't be able to access them outside the EU</p>
+          </div>
+
+          <div class="question" id="section5">What happens if I forget my password?<span class="arrow">+</span></div>
+          <div class="answer">
+          <p> Use the password reset option on the login page to reset your password. You will need access to the email address you originally used to log in.</p>
+          </div>
+
+          <div class="question" id="section6">How do I access Works-In-Progress?<span class="arrow">+</span></div>
+          <div class="answer">
+          <p> Links to attend this event will be emailed one hour prior to the event start time.</p>
+          </div>
+
+        </section>
+
+
+    <!-- 2 of 2 content-box, this is sign up form -->
+        <sectionc class="content-box">
+
+            <div class="headline"><h1>Register</h1></div>
+
+            <form class="signup-form" action="http://httpbin.org/anything" method="POST">
+           
+                    <label> First Name: 
+                       <input
+                       type="text"
+                       class="input-field"
+                       name="first-name"
+                       id="first-name"
+                       required
+                       placeholder="Mads"
+                        autocomplete="off">
```

Comment:
I think you should have a "/ " in the end after all the attributes because input tag is a self-closing tag.  <input />